warehouse/models/staging/gtfs/gtfs_quality/stg_gtfs_rt__service_alerts_validation_notices.sql

```diff
@@ -0,0 +1,30 @@
+WITH raw_validation_notices AS (
+    SELECT * FROM {{ source('external_gtfs_rt', 'service_alerts_validation_notices') }}
+),
+
+stg_gtfs_rt__service_alerts_validation_notices AS (
+    SELECT
```

Comment:
is there any chance that it's worth having a macro for these columns to ensure consistency between the RT tables? I think I did that for the raw outcomes, not sure our opinion stylistically/organizationally.

Reply:
Generally I try to avoid un-parameterized abstractions, i.e. functions/whatever without arguments. More macros will also increase our dbt project compilation time which isn't too bad right now but can get annoying over time. I don't feel a huge need to macro these 3 tables right now, but maybe if we have more versions of the notice schema in the future?

Reply:
I guess for me, the biggest headache in reviewing PRs that touch RT is trying to check if we actually updated all the locations. So that's why I'd like some kind of abstraction. I am ok to not have it but that's where I am coming from and why I did it for the outcomes.

Reply:
Fair enough, I will adjust